### PR TITLE
Add table comments

### PR DESCRIPTION
**Pull request by strk**
See #23 and #15

### CODE REVIEW

**Comment by strk** [comment]
It looks like the comment is not added if a "bde" schema already exists, so either I do this with "patches" (dbpatch based) or I have to review/modify the `02-bde_schema.sql` code. What do you think @palmerj ?

**Comment by strk** [comment]
Looking at LDS it turns out that table is still being updated (in chunks of 68 records) for some reason, so more information is needed before tagging it as "deprecated"

**Comment by palmerj** [comment]
> It looks like the comment is not added if a "bde" schema already exists, so either I do this with "patches" (dbpatch based) or I have to review/modify the 02-bde_schema.sql code. What do you think @palmerj ?

I would say add it to the patches.

File: sql/02-bde_schema.sql [hunk 1]
```diff
@@ -3773,4 +3773,5 @@ GRANT SELECT ON TABLE cbe_title_parcel_association TO bde_user;
 END;
 $SCHEMA$;
 
+COMMENT ON TABLE bde.crs_statist_area IS 'DEPRECATED: Territorial Authority names recorded in Landonline';
```
Comment (palmerj) [inline sql/02-bde_schema.sql]: `Territorial Authority` wrong table name?

**Comment by strk** [comment]
I've updated the comment with full detail of where the information came from.

**Comment by strk** [comment]
New approach: comments in their own file sql/06-schema_comments.sql
I'm now looking at how to possibly automate creation of that file from the PDF file.

**Comment by strk** [comment]
So while I wait for the official source of documentation (#54) I've pushed a perl script that extracts table description from a `links` rendering of the HTML generated by `pdf2htmlEX` with `LDS Full LandOnLine Data Dictionary and Models` PDF a input. Convoluted generation, anyway in this PR you have both the tool *and* the output, so nothing to worry too much about.

One issue with this approach though is that some of the tables documented in the LDS PDF are _not_ present in this `bde` schema, but rather are installed by `linz-lds-bde-schema` under the `bde_ext` schema. I'll tweak the script to omit those tables.

**Pull request merged**